src/main/fc/cli.c

```diff
@@ -1080,15 +1079,18 @@ static void printValuePointer(const clivalue_t *var, const void *valuePointer, u
 /* not currently used
```

Comment:
rather then comment out the code, surround it with `if (false)` since then it will still be compiled and optimized out.  if it's really not needed then delete it - we use git SCM.